### PR TITLE
chore: update readme

### PR DESCRIPTION
**Pull request by manato**
## PR Type

- Improvement

## Description

This PR just modifies the contents on README.

## Review Procedure

I would like reviews to check the modified README contains appropriate description.

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [x] Commits are properly organized and messages are according to the guideline
- [x] (Optional) Unit tests have been written for new behavior
- [x] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [x] All open points are addressed and tracked via issues or tickets

## CI Checks

- **Build and test for PR**: Required to pass before the merge.

### CODE REVIEW

**Review by drwnz — APPROVED** [review]
LGTM

**Comment by wep21** [comment]
@drwnz please check #12 before merging this PR.

**Review by drwnz — CHANGES_REQUESTED** [review]
@manato Since these have now been handled by a different PR (merged), can you please remove the commit c9820a5? Then I will merge this PR. Thanks!

**Comment by wep21** [comment]
@drwnz @manato I rebased the branch.

File: README.md
Comment (wep21) [inline README.md]: backslash is necesarry?

**Comment by wep21** [comment]
build and test for galactic is failed due to the version of `cpplint`, `clang-format`, so we can once ignore this failure.

File: README.md
Comment (manato) [reply README.md]: @wep21 I appreciate your suggestion. As the backslashes you pointed out are surely unnecessary, I made a commit to remove them.

**Review by wep21 — APPROVED** [review]
LGTM

**Comment by manato** [comment]
@drwnz @wep21 
Thank you very much for your time. I think the problems are gone, so I appreciate it if you guys approve this PR if there is no problem left.

**Review by drwnz — APPROVED** [review]
LGTM

**Pull request merged**
